### ISSUE TITLE
Size mismatch between downloaded versions.

### ISSUE THREAD

**Issue by TricksterSam** [benirose/O_C-BenisphereSuite#72]
I have a version of O_C-Benispheresuite-BSSv1.3.1 that I modified, compiled, and installed, but moving my one change (Runglbook) into the repo I cloned from Github results in an out-of-memory error when compiling.

`region FLASH overflowed by 28108 bytes
collect2: error: ld returned 1 exit status
Error compiling for board Teensy 3.2 / 3.1.`

I'm kind of new to forking other people's projects. Can I run some kind of diff on the folder to find out where those extra kilobytes are?

**Comment by djphazer** [comment]
What are you using to compile? I'm afraid we've abandoned support for the Arduino IDE in favor of Platform IO, but you might still be able to make it work if you use the "smallest code" compiler optimization.

**Comment by TricksterSam** [comment]
I was using method B for installing Ornament and Crime which is linked to from the front page. I compiled it with Arduino IDE 1.8.1 and installed it with Teensyduino v1.35. Arduino 1.8.1 is now giving me the error. I did have to make one change to include USB MIDI in the preferences.

I wasn't aware of Platform IO. Looks useful because I prefer to work in VSCodium. Will work on getting C++ working with VSCodium first. Thanks!
